### PR TITLE
Add Svelte docset

### LINKED ISSUE #2733: request: svelte
- https://svelte.dev/
- https://svelte.dev/docs

### PR DESCRIPTION
**Pull request by noahlh**
Based on the latest version of the docs (aligns with v3.23.2 of the library).

Resolves #2733

### CODE REVIEW

File: docsets/Svelte/docset.json [hunk 1]
```diff
@@ -0,0 +1,16 @@
+{
+  "name": "Svelte",
+  "version": "3.23.2",
+  "archive": "Svelte.tgz",
```
Comment (Kapeli) [inline docsets/Svelte/docset.json:4]: You need to add the archive at this path

**Comment by Kapeli** [comment]
Please don't delete the Sample Docset

**Comment by noahlh** [comment]
@Kapeli Oops sorry about that!  Misunderstood the instructions.  Fixed both issues.

**Comment by Kapeli** [comment]
Thank you! 👍

**Pull request merged**